model/change_point_detection.go

```diff
@@ -428,3 +461,18 @@ func createChangePoint(ctx context.Context, env cedar.Environment, resultToUpdat
 	_, err := env.GetDB().Collection(perfResultCollection).UpdateOne(ctx, filter, update)
 	return errors.Wrap(err, "Unable to create change point")
 }
+
+func TriageChangePoint(ctx context.Context, env cedar.Environment, perfResultId string, measurement string, status TriageStatus) error {
+	filter := bson.M{
+		perfIDKey: perfResultId,
+		bsonutil.GetDottedKeyName(perfAnalysisKey, perfAnalysisChangePointsKey, perfChangePointMeasurementKey): measurement,
+	}
+	update := bson.M{
+		"$set": bson.M{
+			bsonutil.GetDottedKeyName(perfAnalysisKey, perfAnalysisChangePointsKey, "$", perfChangePointTriageKey, perfTriageInfoStatusKey):    status,
+			bsonutil.GetDottedKeyName(perfAnalysisKey, perfAnalysisChangePointsKey, "$", perfChangePointTriageKey, perfTriageInfoTriagedOnKey): time.Now(),
+		},
+	}
+	_, err := env.GetDB().Collection(perfResultCollection).UpdateOne(ctx, filter, update)
```

Comment:
you have to check the result (first returned value). the error value is only non-nil when there's a connection, parsing, or other related error, if you failed to find or update a document, that's in the result.